src/utils/daos_hdlr.c

```diff
@@ -2442,13 +2439,47 @@ cont_clone_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
+dfuse_count_query(struct cmd_args_s *ap)
+{
+	struct dfuse_mem_query query = {};
+	int                    rc    = -DER_SUCCESS;
+	int                    fd;
+
+	fd = open(ap->path, O_NOFOLLOW, O_RDONLY);
+	if (fd < 0) {
+		rc = errno;
+		DH_PERROR_SYS(ap, rc, "Failed to open path");
+		return daos_errno2der(rc);
+	}
+
+	query.ino = ap->dfuse_mem.ino;
+
+	rc = ioctl(fd, DFUSE_IOCTL_COUNT_QUERY, &query);
+	if (rc < 0) {
+		rc = daos_errno2der(errno);
+		DH_PERROR_DER(ap, rc, "ioctl failed");
+		goto close;
+	}
```

Comment:
Should these error prints be consistent? The first prints an `errno` and the second prints a `D_ERR`

Reply:
Yes they should.

Reply:
Somehow you're commenting on an old commit within the PR but yes, I've updated all these to be consistent and fixed the `query` command as well.